### PR TITLE
new userspace for @ibnuda

### PR DESCRIPTION
**Pull request by ibnuda**
created a user space to ease maintenance for some boards that i use.

## Description

- created user space for ibnuda.
- changed ibnuda's atreus keymap.
- added ibnuda's dactyl manuform 4x5 keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/ibnuda/config.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#define COMBO_COUNT 18
+#define COMBO_TERM 100
+
+#define IGNORE_MOD_TAP_INTERRUPT
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+#define TAPPING_TERM 200
```
Comment (ridingqwerty) [inline users/ibnuda/config.h]:
```suggestion
```

`#define TAPPING_TERM 200` is here twice.

File: keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/config.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/* Use I2C or Serial, not both */
+
+#define USE_SERIAL
+// #define USE_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
```
Comment (ridingqwerty) [inline keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/config.h]: Did you mean to include this file?  It's the only one pertaining to a dactyl_manuform.

Comment (ibnuda) [reply keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/config.h]: yes, the default is `#define EE_HANDS`.

File: users/ibnuda/config.h (hunk 1 above)
Comment (ibnuda) [reply users/ibnuda/config.h]: you're right. it was past midnight when i commit the file.

File: keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/config.h (hunk 2 above)
Comment (ridingqwerty) [reply keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/config.h]: Ok just checking, thanks.

File: keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/keymap.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#include QMK_KEYBOARD_H
+
+#include "action_layer.h"
+#include "eeconfig.h"
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5/keymaps/ibnuda/keymap.c]: the `action_layer.h` shouldn't be needed, and I think the eeconfig.h isn't either, but not as sure on that one.

File: users/ibnuda/ibnuda.c [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#include "quantum.h"
```
Comment (drashna) [inline users/ibnuda/ibnuda.c]: This isn't needed, actually. 

```suggestion
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**